Cargo.lock

```diff
@@ -1255,22 +1255,22 @@ dependencies = [
 
 [[package]]
 name = "assets-common"
-version = "0.18.0"
+version = "0.18.1"
```

Comment:
To be exact 2412 doesn't actually use assets-common 0.18.1, it only uses 0.7.0.

Reply:
Yep, Cargo.lock should not be here, I reverted it back to the state how it was in master